skunkworks_crow/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <!-- Bluetooth Related -->
```

Comment:
The permissions are self explanatory. No need for this comment

Reply:
Fixed.